src/less/chat.less

```diff
@@ -23,6 +24,118 @@
   }
 }
 
+.chat-card-custom {
+  .chat-card-custom-item{
+    img{
+      margin:5px;float:left;
+      border:none;
+      border-radius: 10px;
+      width:30px;
+    }
+
+    p span{
+      display: block;
+      color: black;
+      font-weight: bold;
+    }
+
+    p button {
+      margin: 5px 0px 5px 0px;
+      background-color: indianred;
+      border-color: #000000;
+      color: #FFFFFF;
+      font-size: 12px;
+      height: 20px;
+      width: 60px;
+      line-height: inherit;
+    }
+
+    p button.chat-button-save{
+      margin: 5px 0px 5px 0px;
+      background-color: grey;
+      border-color: black;
+      color: #FFFFFF;
+      font-size: 12px;
+      height: 20px;
+      width: 90px;
+      line-height: inherit;
+    }
+
+    h2{
+      position:initial;
+    }
+  }
+
+  div.chard-card-custom-token{
+    img{
+      float:left;
+      border:none;
+      width:60px;
+    }
```

Comment:
See above comments about proper spacing between properties and values.

Reply:
done